src/app.ts

```diff
@@ -99,6 +99,45 @@ async function app() {
 
   logger.info(`Starting ${getVersion()}`);
 
+  logger.info('Arguments:');
+  if (programOptions.logLevel) {
+    logger.info(`  --log-level ${programOptions.logLevel}`);
+  }
+  if (programOptions.dependency) {
+    logger.info(`  --dependency ${programOptions.dependency}`);
+  }
+  if (programOptions.style) {
+    logger.info(`  --style ${programOptions.style}`);
+  }
+  if (programOptions.fshingTrip) {
+    logger.info('  --fshing-trip');
+  }
+  if (programOptions.installedSushi) {
+    logger.info('  --installed-sushi');
+  }
+  if (programOptions.fileType) {
+    logger.info(`  --file-type ${programOptions.fileType}`);
+  }
+  if (programOptions.indent) {
+    logger.info('  --indent');
+  }
+  if (programOptions.metaProfile) {
+    logger.info(`  --meta-profile ${programOptions.metaProfile}`);
+  }
+  if (!programOptions.aliasFile) {
```

Comment:
Sorry I just noticed this, but the negation go added to the wrong `if` statement. This needs to be on the `alias` statement, not the `aliasFile` statement. Can you change this line back to just say `if (programOptions.aliasFile)` and add the negation to the one below?

Reply:
Done.